gammapy/data/gti.py

```diff
@@ -438,6 +438,56 @@ def union(self, overlap_ok=True, merge_equal=True):
         merged = Table(rows=merged, names=["START", "STOP"], meta=self.table.meta)
         return self.__class__(merged, reference_time=self.time_ref)
 
+    def intersect(self, other, padding=0.0):
```

Comment:
It might be confusing that union applies only to self while intersection requires an other, could `other` be optional for both union and intersection ?